src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -450,6 +450,12 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'type': 'boolean',
 			'default': false,
 			'description': localize('queryEditor.promptToSaveGeneratedFiles', "Prompt to save generated SQL files")
+		},
+		'queryEditor.executionPlan.expensiveOperationMetric': {
+			'type': 'string',
+			'enum': ['off', 'actualElapsedTime', 'actualElapsedCpuTime', 'cost', 'subtreeCost', 'actualNumberOfRowsForAllExecutions', 'numberOfRowsRead'],
```

Comment:
do also need an `enumDescriptions` array? if these identifiers will be user facing we probably do.